sdk/typescript/src/providers/provider.ts

```diff
@@ -283,6 +285,10 @@ export abstract class Provider {
    */
   abstract dryRunTransaction(txBytes: string): Promise<TransactionEffects>;
 
+  abstract getDynamicFields(parent_object_id: ObjectId, cursor: ObjectId | null, limit: number | null): Promise<DynamicFieldPage>;
+
+  abstract getDynamicFieldObject(parent_object_id: ObjectId, name: string): Promise<GetObjectDataResponse>;
+
```

Comment:
Can we add docs comments for these two methods? Feel free to reference the RPC comments